pkg/virt-controller/services/template_test.go

```diff
@@ -1517,14 +1517,19 @@ var _ = Describe("Template", func() {
 				Expect(ok).To(BeTrue())
 				Expect(int(tun.Value())).To(Equal(1))
 
-				found := false
+				foundNetAdmin := false
+				foundNetRaw := false
 				caps := pod.Spec.Containers[0].SecurityContext.Capabilities
 				for _, cap := range caps.Add {
```

Comment:
I think that we could use something like

```
Expect(caps.Add).Should(ConsistOf(CAP_NET_ADMIN, CAP_NET_RAW))
```

Reply:
See my update. Since we also have CAP_SYS_NICE in the mix, I used ContainElement - but I removed the for loop and the boolean test variables. 

I think the original implementer did this because they didn't realize you actually have to cast the constants to kubev1.Capability in order to get the matcher to work.